### PR TITLE
Init SIWS doc pages

### PR DESCRIPTION
**Pull request by spacesailor24**
- New pages:
    - [Authenticating Sign-in With Solana Messages](https://docs-main-v2-git-wyatt-sws-accs-lit-protocol.vercel.app/sdk/authentication/authenticating-siws)
    - Generating Session Sigs [Using a Sign-in With Solana Message](https://docs-main-v2-git-wyatt-sws-accs-lit-protocol.vercel.app/sdk/authentication/session-sigs/siws-pkp-session-sigs)
    - [Sign-in With Solana Access Control](https://docs-main-v2-git-wyatt-sws-accs-lit-protocol.vercel.app/sdk/access-control/solana/siws-access-control)

Refactored the intro to (and move the page location): Solana [Access Control Conditions Examples](https://docs-main-v2-git-wyatt-sws-accs-lit-protocol.vercel.app/sdk/access-control/solana/sol-rpc-conditions)

### CODE REVIEW

File: docs/sdk/access-control/solana/overview.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Solana Based Access Control
+
```
Comment (a1ttech) [inline docs/sdk/access-control/solana/overview.md:2]: I might just add a sentence at the beginning that quickly summarizes what this guide will be covering. Something like "This guide demonstrates how you can use Lit to permit [decryption](https://developer.litprotocol.com/sdk/access-control/intro) rights to specific addresses on Solana"

File: docs/sdk/access-control/solana/siws-access-control.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
+# Sign-in With Solana Access Control
+
+As covered in the [overview](./overview.md), the Lit nodes currently have limited built-in support for authenticating signed Solana messages. They support deriving the Solana public key from signed data, but they do **not** perform any validation on what was signed. This means that any data signed by a specific Solana wallet will be accepted, which opens the door for signature malleability and replay attacks.
+
+This guide covers how to implement Sign-in With Solana (SIWS) authentication to provide access control using Lit Actions and Phantom's [SIWS specification](https://github.com/phantom/sign-in-with-solana/tree/main).
```
Comment (a1ttech) [inline docs/sdk/access-control/solana/siws-access-control.md:5]: "... allowing you to permit access to content encrypted with Lit to users on Solana"

File: docs/sdk/access-control/solana/overview.md (hunk 1 above)
Comment (spacesailor24) [reply docs/sdk/access-control/solana/overview.md:2]: This page is not specific to any guide, it's regarding ACCs for Solana in general

**Comment by awisniew207** [comment]
Are we alright with the `swimlanes.io` watermark on the images?

File: docs/sdk/authentication/authenticating-siws.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
+# Authenticating Sign-in With Solana Messages
+
+Deriving a Solana public key (aka. address) from a signed message works slightly differently than when using Lit to authenticate signed messages on EVM chains.
+
+Currently the Lit nodes support deriving an address from a Sign-in With Ethereum Message (EIP-5573), while also validating what was signed using the [EIP-5573 specification](https://eips.ethereum.org/EIPS/eip-5573) (eg. the signed message hasn't expired and has the correct format).
```
Comment (awisniew207) [inline docs/sdk/authentication/authenticating-siws.md:5]:
```suggestion
Currently the Lit nodes support deriving an address from a Sign-in With Ethereum Message (EIP-5573), while also validating what was signed using the [EIP-5573 specification](https://eips.ethereum.org/EIPS/eip-5573) (e.g. the signed message hasn't expired and has the correct format).
```

**Review by awisniew207 — APPROVED** [review]
Looks great!

**Pull request merged**